deploy/helm/charts/templates/priority-class.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: openebs-csi-controller-critical
+  name: openebs-cstor-csi-controller-critical
 value: 900000000
 globalDefault: false
 description: "This priority class should be used for the CStor CSI driver controller deployment only."
 ---
 apiVersion: scheduling.k8s.io/v1
 kind: PriorityClass
 metadata:
-  name: openebs-csi-node-critical
+  name: openebs-cstor-csi-node-critical
```

Comment:
@shovanmaity -- Similar comments. See https://github.com/openebs/jiva-operator/pull/72#discussion_r625763700

Reply:
Again, not a blocker for merge.